job-service-admin/src/main/java/com/hpe/caf/services/admin/HealthCheck.java

```diff
@@ -42,10 +42,11 @@
 
     private final static Logger LOG = LoggerFactory.getLogger(HealthCheck.class);
 
+    private static boolean unhealthyWorker = false;
```

Comment:
As discussed, use return values so that you don't need this `static`.

Reply:
Adjusted in the latest commit.